content/hi/docs/setup/best-practices/enforcing-pod-security-standards.md

```diff
@@ -0,0 +1,94 @@
+---
+समीक्षक:
+- लंबा क्लेयर
+- लिगिट्ट
+शीर्षक: पॉड सुरक्षा मानकों को लागू करना
+वजन: 40
+---
+
+<!-- सिंहावलोकन ->
+
+ जब यह लागू करने की बात आती है तो यह पृष्ठ सर्वोत्तम प्रथाओं का अवलोकन प्रदान करता है
+ [पॉड सुरक्षा मानक] (/ डॉक्स/अवधारणाएं/सुरक्षा/पॉड-सुरक्षा-मानक)।
+
+<!-- शरीर -->
+
+## अंतर्निहित पॉड सुरक्षा प्रवेश नियंत्रक का उपयोग करना
+
+{{< फीचर-स्टेट for_k8s_version="v1.22" State="alpha" >}}
+
+[पॉड सुरक्षा प्रवेश नियंत्रक](/ डॉक्स/संदर्भ/एक्सेस-ऑथन-ऑथज़/प्रवेश-नियंत्रक/#पॉडसिक्योरिटी)
+पदावनत PodSecurityPolicies को बदलने का इरादा रखता है।
```

Comment:
when localizing, I'd consider writing borrowed Kubernetes terms like this in their singular form:
```suggestion
पदावनत PodSecurityPolicy को बदलने का इरादा रखता है।
```
however, if that really doesn't feel idiomatic, it's OK to retain the plural.